### PR TITLE
EVO-388: Create a shortcut to access gravitons app/console in wrapper context

### PR DESCRIPTION
**Pull request by PanteraLX**
When we want to access gravitons app/console in a wrapper context, we have to dig deep in the wrappers vendor folder. This PR should create a link to the console file in vendor/bin (the link is named graviton).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,8 +12,7 @@ web/explorer
 mongodata/
 src/GravitonDyn/*
 app/config/parameters.yml
-app/config/parameters_local.xml
-bin
+app/config/parameters_local.xmlgit
```
Comment (lapistano) [inline .gitignore]: typo?

File: bin/graviton [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env php
+<?php
+
+$path = __DIR__ . '/../app/console';
```
Comment (lapistano) [inline bin/graviton]: sure this works? 
this script will be copied in to __DIR__/vendor/bin and the console script will be located in __DIR__/vendor/graviton/graviton/bin ... I do not see where the $path variable does take this into account. pls verify in a wrapper situation.

Comment (PanteraLX) [inline bin/graviton]: As far as I know is vendor/bin/graviton only a simlink to vendor/graviton/graviton/bin/graviton. I tested it in wrapper context and it worked

Comment (lapistano) [inline bin/graviton]: nice .. merci

**Pull request merged**